src/ImageViewer/resources/translations/imageviewer_zh_TW.ts

```diff
@@ -0,0 +1,1631 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="zh_TW">
+<context>
+    <name>AboutDialog</name>
+    <message>
+        <location filename="../../src/GUI/Dialogs/AboutDialog.cpp" line="183"/>
+        <source>Peter S. Zhigalov</source>
+        <translation>jackusay</translation>
```

Comment:
This is translation of author's name. Contributors list is [stored separately](https://github.com/AlienCowEatCake/ImageViewer/blob/1687e991f652b096264b6ce23af1e58acf05dd8b/src/ImageViewer/src/GUI/Dialogs/Contributors.h) and updated from git history as part of release process. English version "Peter S. Zhigalov" should be here

Reply:
done